src/_guides/packages.md

```diff
@@ -0,0 +1,227 @@
+---
+title: How to use packages
+short-title: Packages
+description: Learn more about pub, Dart's tool for managing packages.
+---
+
+The Dart ecosystem uses _packages_ to manage shared software
+such as libraries and tools.
+To get Dart packages, you use the **pub package manager**.
+You can find publicly available packages on the [**Pub site**,]({{site.pub}})
+or you can load packages from the local file system or elsewhere,
+such as git repositories.
+Wherever your packages come from, pub manages version dependencies,
+helping you get package versions that work with each other and
+with your SDK version.
+
+Most [Dart-savvy IDEs][] offer support for using pub that
+includes creating, installing, updating, and publishing packages.
+Or you can use [`pub` on the command line](/tools/pub/cmd).
+
+{% include flutter-packages.md %}
+
+At a minimum,
+a Dart package is a directory containing a [pubspec file](/tools/pub/pubspec).
+The pubspec contains some metadata about the package. Additionally,
+a package can contain dependencies (listed in the pubspec),
+Dart libraries, apps, resources, tests, images, and examples.
+
+<aside class="alert alert-info" markdown="1">
+  **Note:** If your Dart app depends on one or more packages,
+  then your app itself must be in a package.
+</aside>
+
+To use a package, do the following:
+
+* Create a pubspec (a file that lists package dependencies and includes
```

Comment:
s/pubspec/`pubspec.yaml`/

Reply:
This would be a pretty invasive change, and we have a policy of not referring to files by name. (Witness the packages dir/file fun and the name change of the analysis options file.) But I will clarify the name here and keep an eye out for other places.